ir/memory.cpp

```diff
@@ -1443,7 +1443,7 @@ Memory::Memory(State &state) : state(&state), escaped_local_blks(*this) {
     return;
 
   next_nonlocal_bid
```

Comment:
the function that updates this needs tweaking as well I think (`nextNonlocalBid`)